com.unity.render-pipelines.universal/Documentation~/lit-shader.md

```diff
@@ -50,6 +50,7 @@ The __Surface Inputs__ describe the surface itself. For example, you can use the
 | __Normal Map__              | Adds a normal map to the surface. With a [normal map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterNormalMap.html?), you can add surface details like bumps, scratches and grooves. To add the map, click the object picker next to it. The normal map picks up ambient lighting in the environment. <br/>The float value next to the setting is a multiplier for the effect of the  __Normal Map__. Low values decrease the effect of the normal map. High values create stronger effects. |
 | __Occlusion Map__           | Select an [occlusion map](https://docs.unity3d.com/Manual/StandardShaderMaterialParameterOcclusionMap.html).  This simulates shadows from ambient light and reflection, which makes lighting look more realistic as less light reaches corners and crevices of objects. To select the occlusion map, click the object picker next to it. |
+| __Clear Coat Map__          | Adds a second material layer which simulates a transparent and thin coating on top of the base material. The feature is enabled by checking the Clear Coat checkbox. Setting the strength to 0 does not disable the feature. Clear coat has roughly twice the cost of a base material as lighting is evaluated once per layer. Clear coat supports effect strength (or intensity) and smoothness features. It's IOR is fixed to 1.5. The clear coat can be controlled using a map where the red channel represents the stregth property and the green channel represents the smoothness property. If a map is provided then the constant values for strength and smoothness are used as a multiplier for the map's per pixel values. The clear coat affects the color and the smoothness values of the underlying base material.|
```

Comment:
update docs to replace `strength` with `mask` and to remove mention of smoothness.

Reply:
done